Library/Homebrew/formula.rb

```diff
@@ -127,6 +127,11 @@ class Formula
   # @private
   attr_accessor :local_bottle_path
 
+  # When performing a build, test, or other loggable action, indicates which
+  # log file location to use.
```

Comment:
Well, a file name is still a location, right? ;) And I still think of it is a "type" rather than just a prefix; the prefix is derived from it.

The values are unconstrained and unchecked now, and you could stick any (valid file name substring) value in there. Could enumerate the ones the rest of the code uses, but if we want to do that, maybe it should go back to being a constrained symbol type? It feels a little un-Rubyish to document a limit that's just the state of current usage by the rest of the code.